main_test.go

```diff
@@ -97,6 +97,13 @@ func TestGoTorchCommand(t *testing.T) {
 	mockVisualizer.AssertExpectations(t)
 }
 
+func TestGoTorchCommand(t *testing.T) {
+	testGoTorchCommand(t, "http://localhost")
+
+	// Trailing slash in url should still work.
+	testGoTorchCommand(t, "http://localhost/")
```

Comment:
Yep, table tests are more common, but parameterized tests are also common, e.g.
https://github.com/golang/go/blob/master/src/unicode/utf8/utf8_test.go#L222

In general, we should avoid code duplication, as long as the test is still readable and obvious.